Samples/MyTestiOSApp/MyTestiOSApp/ADTestAppAcquireTokenViewController.m

```diff
@@ -28,7 +28,15 @@
 #import "ADTestAppProfileViewController.h"
 #import "ADTestAppClaimsPickerController.h"
 
-@interface ADTestAppAcquireTokenViewController () <UITextFieldDelegate>
+#if AD_MAM_SDK_TESTING
+#import <IntuneMAMWalledGarden/IntuneMAM.h>
+#endif
+
+@interface ADTestAppAcquireTokenViewController () <UITextFieldDelegate
```

Comment:
nit: this looks a bit awkward, can we have whole protocol definite in if and else

Reply:
done